.github/workflows/native-image-tests.yml

```diff
@@ -39,9 +39,11 @@ jobs:
           jvm: temurin:1.11
 
       - name: Gather version
+        # some cleanup of the sbt output to get the version sbt will use when publishing below
         run: |-
-          echo `git describe --tags | sed -e "s/v\(.*\)-\([0-9][0-9]*\).*/\\1+\\2-/"``git rev-parse HEAD | head -c8`-SNAPSHOT > ~/.version
-          cat ~/.version
+          sbt "core/version" --batch --no-colors | tail -n 1 | cut -f 2 -d ' ' | tr -d '\n' > ~/.version
```

Comment:
Or using sbt's `print`:

```
sbt "print core/version" | tail -n 1 | tr -d '\n' > ~/.version
```

Reply:
Oh, don't remember seeing that before, neat.